web_widget_bokeh_chart/static/src/less/custom-bokeh.less

```diff
@@ -0,0 +1,32 @@
+.bk-root {
+    /* Overriding this styling option Odoo menu is now displayed on top of bokeh graphs */
+    z-index: 0;
+
+    /* Overriding this styling options the graphs are always displayed inside their container */
+    .bk-layout-stretch_both {
+        height: inherit !important;
```

Comment:
It would be good to include a comment that justifies that all of these `!important` sentences are needed to override inline styles hardcoded by the underlying library. Just because this is not very recommended in general.

Reply:
done